R/swimp.R

```diff
@@ -114,3 +166,6 @@ swimp <-
     }
     ans
   }
+
+#' @import utils
+utils::globalVariables(c("swimp_core"))
```

Comment:
Is this needed?

Reply:
Please see similar comments re: global variables and R CMD check.